pkg/api/pod/warnings.go

```diff
@@ -160,6 +160,9 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		if v.Ephemeral != nil && v.Ephemeral.VolumeClaimTemplate != nil {
 			warnings = append(warnings, pvcutil.GetWarningsForPersistentVolumeClaimSpec(fieldPath.Child("spec", "volumes").Index(i).Child("ephemeral").Child("volumeClaimTemplate").Child("spec"), v.Ephemeral.VolumeClaimTemplate.Spec)...)
 		}
+		if v.CephFS != nil {
+			warnings = append(warnings, fmt.Sprintf("%s: deprecated in v1.28, non-functional in v1.29+", fieldPath.Child("spec", "volumes").Index(i).Child("CephFS")))
+		}
```

Comment:
I think we normally need to wait for 3 releases after deprecation before we can remove the code.  GlusterFS is a special case.